app/steps/ui/summary/template.html

```diff
@@ -34,9 +34,11 @@ <h2 class="heading-medium">{{content.Summary.deceasedHeading}}</h2>
     <dl class="check-your-answers check-your-answers--long">
         {% include "ui/summary/includes/deceased.njk" %}
     </dl>
-
-    <p><img class="icon-file-download" alt="Download PDF document"/>&nbsp;
-    <a id="checkAnswerHref" href="/checkAnswersPdf">{{content.Summary.checkAnswersPdf}}</a></p>
+    <p>
+        <img src="{{ asset_path }}images/icon-file-download.png?" alt="Download pdf" width="30" height="30"/>
```

Comment:
We don't need to add this image manually as the GDS toolkit has a class that can add it so we should be using that instead

Reply:
yes, template has already been changed to use